tests/GitObjectDb.ModelCodeGeneration.Tests/GitObjectDb.ModelCodeGeneration.Tests.csproj

```diff
@@ -31,6 +31,10 @@
     <PackageReference Include="Microsoft.Build.Locator" Version="1.1.2" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="15.9.20" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.9.20" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="2.6.3">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
```

Comment:
doesn't PrivateAssets take priority over IncludeAssets ? everything will be consumed and won't flow to the dependencies, so I don't think the IncludeAssets line is necessary